packages/spectral/src/generators/componentManifest/getInputs.ts

```diff
@@ -87,9 +87,24 @@ export const INPUT_TYPE_MAP: Record<InputFieldDefinition["type"], ValueType> = {
   dynamicFieldSelection: "string",
 };
 
+const ESCAPED_REPLACEMENTS: Record<string, string> = {
```

Comment:
I attempted a ton of different regex with replacement groups here, however a direct match was the only thing that worked.